src/sampling.jl

```diff
@@ -273,9 +284,25 @@ end
 
 
 ### Interface functions (poly-algorithms)
+"""
+    sample(a, [wv::WeightVec])
 
+Select a single random element of `a`. Weighted sampling is performed if
```

Comment:
Instead of or in addition to "weighting sampling", maybe say something about the probability of each element being proportional to its weight? Same for functions below.